### DISCUSSION TITLE
cabin press on ground

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#6079] [General]
Hi to all,

I flew a leg tofay and ended in MMMY. Was on blocks with running APU. After switching off ENG 1+2, i got a message on my ECAM which I didn´t really understand. Bug?

![grafik](https://user-images.githubusercontent.com/76531893/138691236-6a440108-c46c-410e-b8ee-9787d5426fb5.png)

Greets Klaus

**Comment by Tigerhawk63** [comment]
I got that after my last flight too. Turning off the packs stopped the message.

**Reply by CptBayer** [reply]
that´s what the ECAM told you to do :-P ....

**Comment by Benjozork** [comment]
Fixed in master, experimental soon
